packages/components/src/styles/footnotes/footnotes.stories.ts

```diff
@@ -55,39 +52,42 @@ export const Default = {
 };
 
 /**
- * Use the class with a `ul` element to display the footnotes as an unordered list.
+ * Use the alternative html elements for alternative appearances:
+ *
+ * - unnumbered: use a `ul` element to create an unnumbered list.
+ * - numbered: use an `ol` element to create a numbered list.
+ * Usually start with "1", use the html `start` attribute to set the starting number of the list.
  */
 
-export const UnorderedLists = {
-  parameters: { controls: { include: [] } },
-
-  render: (args: any) => {
-    return generateTemplate({
-      options: {
-        templateContent: '<div class="sd-container"><ul class="%CLASSES%">%SLOT%</ul></div?'
-      },
-      args
-    });
-  }
+export const Variants = {
+  render: () => html`
+    <div class="flex flex-col gap-12">
+      <ul class="sd-footnotes">
+        <li>Lorem ipsum dolor sit amet, consetetur sadipscing elitr.</li>
+        <li>Sed diam nonumy eirmod tempor invidunt ut labore.</li>
+        <li>Dolore magna aliq erat, sed diam voluptua.</li>
+      </ul>
+      <ol start="150" class="sd-footnotes">
+        <li>Lorem ipsum dolor sit amet, consetetur sadipscing elitr.</li>
+        <li>Sed diam nonumy eirmod tempor invidunt ut labore.</li>
+        <li>Dolore magna aliq erat, sed diam voluptua.</li>
+      </ol>
+    </div>
+  `
 };
 
 /**
- * This sample shows how to use the `start` attribute to set the starting number of the list. This is a html standard attribute.
+ * Use the `sd-footnotes--inverted` class to invert the color of the footnotes.
  */
 
-export const Sample = {
-  parameters: { controls: { include: [] } },
-  name: 'Sample: Set start',
-  render: (args: any) => {
-    return generateTemplate({
-      options: {
-        templateContent: `<div class="sd-container">
-        <ol start="50" class="%CLASSES%">%SLOT%</ol>
-        <ol start="150" class="%CLASSES%">%SLOT%</ol>
-        <ol class="%CLASSES%">%SLOT%</ol>
-      </div>`
-      },
-      args
-    });
-  }
+export const Inverted = {
+  render: () => html`
+    <div class="sd-container sd-container--variant-primary">
```

Comment:
Just out of curiosity, why did you use these classes in this example?

Reply:
It was using the container by default. Looks like a better approach considering that padding/others are defined my the container.